azure_functions_worker/constants.py

```diff
@@ -81,5 +81,15 @@
 # Base extension supported Python minor version
 BASE_EXT_SUPPORTED_PY_MINOR_VERSION = 8
 
+# Appsetting to turn on OpenTelemetry support/features
+# Includes turning on Azure monitor distro to send telemetry to AppInsights
 PYTHON_ENABLE_OPENTELEMETRY = "PYTHON_ENABLE_OPENTELEMETRY"
-PYTHON_ENABLE_OPENTELEMETRY_DEFAULT = True
+PYTHON_ENABLE_OPENTELEMETRY_DEFAULT = False
```

Comment:
Since the feature is opt-in and using a python specific app setting, we should add it to our logs. The app setting can be added in `azure_functions_worker/utils/app_setting_manager.py`